Updating contributing.md to make it more clear. Fixes Issue #107

Ethan-bradley opened the pull request:
Fixes Issue #107. This makes the contributing doc more clear and fixes some bugs.
Changes listed below:
Maybe add that estimated time to setup everything (including installing docker) is around 30-45 minutes.
Add Open terminal
Add Install git (May add instructions for git installation: https://docs.gitlab.com/ee/gitlab-basics/start-using-git.html) if not already installed. If you are on a mac, just type in git and hit ok to install software tools.
Run the following commands in terminal:
Change git clone line to git clone https://github.com/openoakland/woeip.git, currently incorrect.
Separated the first two code lines to allow for easier copy and paste.
That make local.shell shoud look like this (outside of shell):
make local.shell
Add go to 0.0.0.0:8080 on your web browser
Put in the username that you put in the terminal for the email.

---
name: Pull Request
about: Request merging changes into the code base
title: ''
labels: ''
reviewers: ''

---

## Checklist
- [ ] Add description
- [ ] Reference open issue pull request addresses
- [ ] Pass linting check
  - complete on the local machine with `make local.shell` `make quality`
- [ ] Pass functional tests
  - complete on the local machine with `make local.shell` `make test`
- [ ] Request code review
  - Please allow **36 hours** from opening a pull request before merging a pull request- even if it has already recieved an approving review.
- [ ] Address comments on code and resolve requested changes
- [ ] Merge own code

## Description
Issue: #

*Brief description of solution*

Ethan-bradley committed:
Updating contributing.md to make it more clear. Fixes Issue #107

Fixes Issue #107. This makes the contributing doc more clear and fixes some bugs.
Changes listed below:
Maybe add that estimated time to setup everything (including installing docker) is around 30-45 minutes.
Add Open terminal
Add Install git (May add instructions for git installation: https://docs.gitlab.com/ee/gitlab-basics/start-using-git.html) if not already installed. If you are on a mac, just type in git and hit ok to install software tools.
Run the following commands in terminal:
Change git clone line to git clone https://github.com/openoakland/woeip.git, currently incorrect.
Separated the first two code lines to allow for easier copy and paste.
That make local.shell shoud look like this (outside of shell):
make local.shell
Add go to 0.0.0.0:8080 on your web browser
Put in the username that you put in the terminal for the email.

Ethan-bradley committed:
Update contributing.md

Fixed some grammar errors.

TangoYankee commented on .github/contributing.md:
```diff
@@ -24,11 +24,17 @@ These are general guidelines on the flow from design to implementation
 ## Code development
+Following the installation steps (including installing docker) takes around 20-35 minutes. 
 
+Make sure you have git installed (Instructions for git installation: https://docs.gitlab.com/ee/gitlab-basics/start-using-git.html if not already installed). 
```
Recommend Markdown link for styling: `([Instructions for git installation](https://docs.gitlab.com/ee/gitlab-basics/start-using-git.html))`

Ethan-bradley replied:
will change the style

TangoYankee commented on .github/contributing.md:
````diff
@@ -50,15 +56,20 @@ When running the service with this command, it will be configured to run using t
 ```
 root@<container ID>:/app/woeip#	./manage.py createsuperuser 
 ```
+Enter a username, password and email. These are for logging into the docker container. 
````
The superuser is for the Django Admin

Ethan-bradley replied:
will add

theecrit replied:
I committed the change directly to your branch; hope that's okay! I figured it was a minor enough change.

Ethan-bradley committed:
Updating contributing.md 

Fixes comments made on the previous pull request commit
Added a markdown link for the git installation instructions and changed wording to (logging into django admin)

theecrit committed:
Corrected link styling

Hope you don't mind me committing this directly; it's such a minor change and I know @Ethan-bradley is unavailable for a few weeks.

Ethan-bradley commented:
That is probably fine

On Tue, Jul 23, 2019 at 9:03 PM Jess Sand <notifications@github.com> wrote:

> *@theecrit* commented on this pull request.
> ------------------------------
>
> In .github/contributing.md
> <https://github.com/openoakland/woeip/pull/114#discussion_r306619667>:
>
> >  ```
>
>  Create a Django superuser:
>
>  ```
>  root@<container ID>:/app/woeip#	./manage.py createsuperuser
>  ```
> +Enter a username, password and email. These are for logging into the docker container.
>
> I committed the change directly to your branch; hope that's okay! I
> figured it was a minor enough change.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openoakland/woeip/pull/114?email_source=notifications&email_token=AHIQ7UVA3JKZ35PQYKHD3FLQA7ICFA5CNFSM4HZFOGRKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOB7LYKSI#discussion_r306619667>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHIQ7UW4G3EGOC5W6AQFB7DQA7ICFANCNFSM4HZFOGRA>
> .
>
